pkg/controllers/collaset/synccontrol/sync_control.go

```diff
@@ -431,9 +475,16 @@ func (r *RealSyncControl) Update(
 	// 3. prepare Pods to begin PodOpsLifecycle
 	podCh := make(chan *PodUpdateInfo, len(podToUpdate))
 	for i, podInfo := range podToUpdate {
+		// The pod is in a "replacing" state, always requires further update processing.
+		if podInfo.isInReplacing {
+			podCh <- podToUpdate[i]
```

Comment:
Is it my misunderstanding that the original Pod will begin to do traffic off before the new Pod is created ?